lib/test_prof/recipes/rspec/let_it_be.rb

```diff
@@ -32,6 +34,19 @@ def register_modifier(key, on: :let, &block)
         LetItBe.modifiers[key] = Modifier.new(on, block)
       end
 
+      def report_duplicates=(value)
+        value = value.to_sym
```

Comment:
What about `report_duplicates = false`? (For example, to temporary disable reporting).

I think, we can remove the value validation altogether and be less defensive here

Reply:
> What about report_duplicates = false? (For example, to temporary disable reporting).

We could omit this value, which would be equiv

Reply:
removed the validation